pkg/virt-controller/services/template.go

```diff
@@ -870,7 +870,7 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 			Resources: k8sv1.ResourceRequirements{
 				Limits: map[k8sv1.ResourceName]resource.Quantity{
 					k8sv1.ResourceCPU:    resource.MustParse("1m"),
-					k8sv1.ResourceMemory: resource.MustParse("12Mi"), // this is the minimum memory limit for cri-o!
+					k8sv1.ResourceMemory: resource.MustParse("40Mi"), // this is the minimum memory limit for cri-o!
```

Comment:
Can you remove the comment please? 12Mi was a real minimum limit accepted by cri-o. If we go beyond that. it is not relevant anymore.

Reply:
Done.